recipes-support/curl/curl/0018-Revert-global_init-ignore-CURL_GLOBAL_SSL-s-absense.patch

```diff
@@ -1,4 +1,4 @@
-From 11858441b7147142ca804bc9af17fd8bb0d06e54 Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Haris Okanovic <haris.okanovic@ni.com>
```

Comment:
Could you update the attribution info, since you modified the .patch contents.

Reply:
Amended.